docs/src/content/docs/hi/guides/sidebar.mdx

```diff
@@ -0,0 +1,541 @@
+
+`label` और `autogenerate` गुणों वाले ऑब्जेक्ट का उपयोग करके एक स्वतः निर्मित समूह जोड़ें। साइडबार प्रविष्टियों के लिए उपयोग करने के लिए आपके `autogenerate` कॉन्फ़िगरेशन को `directory` निर्दिष्ट करना होगा। उदाहरण के लिए, निम्नलिखित कॉन्फ़िगरेशन के साथ:
+
+```js "label:" "autogenerate:"
+starlight({
+	sidebar: [
+		{
+			label: 'मार्गदर्शिका',
+			// 'गाइड' निर्देशिका के लिए लिंक का एक समूह स्वतः उत्पन्न करें।
+			autogenerate: { directory: 'guides' },
+		},
+	],
+});
+```
+
+और निम्न फ़ाइल संरचना:
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+        - advanced/
+          - project-structure.md
+
+</FileTree>
+
+निम्नलिखित साइडबार उत्पन्न होगा:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				{ label: 'अवयवों', link: '/guides/components/' },
+				{ label: 'अंतर्राष्ट्रीयकरण (i18n)', link: '/guides/i18n/' },
+				{
+					label: 'advanced',
+					items: [
+						{ label: 'परियोजना संरचना', link: '/guides/project-structure/' },
+					],
+				},
+			],
+		},
+	]}
+/>
+
+#### फ्रंटमैटर में स्वतः निर्मित लिंक को अनुकूलित करना
+
+स्वचालित रूप से उत्पन्न किए गए लिंक को अनुकूलित करने के लिए अलग-अलग पृष्ठों में [`sidebar फ्रंटमैटर क्षेत्र`](/hi/reference/frontmatter/#sidebar) का उपयोग करें।
+
+साइडबार फ्रंटमैटर विकल्प आपको एक [कस्टम लेबल](/hi/reference/frontmatter/#label) सेट करने या किसी लिंक में एक [बैज](/hi/reference/frontmatter/#badge) जोड़ने, साइडबार से एक लिंक [छिपाने](/hi/reference/frontmatter/#hidden) या एक [कस्टम सॉर्ट भार](/hi/reference/frontmatter/#order) परिभाषित करने की अनुमति देते हैं।
+
+```md "sidebar:"
+---
+# src/content/docs/example.md
+title: मेरा पेज
+sidebar:
+  # लिंक के लिए एक कस्टम लेबल सेट करें
+  label: कस्टम साइडबार लेबल
+  # लिंक के लिए एक कस्टम क्रम सेट करें (निचले अंक ऊपर प्रदर्शित होते हैं)
+  order: 2
+  # लिंक में एक बैज जोड़ें
+  badge:
+    text: नया
+    variant: tip
+---
+```
+
+उपरोक्त फ्रंटमैटर वाले पृष्ठ सहित एक स्वत: उत्पन्न किया गया समूह निम्नलिखित साइडबार उत्पन्न करेगा:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				{ label: 'एक पेज', link: '#' },
+				{
+					label: 'कस्टम साइडबार लेबल',
+					link: '#',
+					badge: { text: 'नया', variant: 'tip' },
+				},
+				{ label: 'एक अन्य पेज', link: '#' },
+			],
+		},
+	]}
+/>
+
+:::note
+`sidebar` फ्रंटमैटर कॉन्फ़िगरेशन का उपयोग केवल स्वतः निर्मित लिंक के लिए किया जाता है और मैन्युअल रूप से परिभाषित लिंक के लिए इसे अनदेखा किया जाएगा।
+:::
+
+## बैजेस
+
+लिंक, समूह और स्वत: उत्पन्न किए गए समूह अपने लेबल के आगे बैज प्रदर्शित करने के लिए `badge` गुण भी शामिल कर सकते हैं।
+
+```js {10,17}
+starlight({
+	sidebar: [
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				// "New" बैज वाला एक लिंक.
+				{
+					label: 'अवयवों',
+					link: '/guides/components/',
+					badge: 'New',
+				},
+			],
+		},
+		// "Deprecated" बैज वाला एक स्वतः निर्मित समूह।
+		{
+			label: 'संदर्भ',
+			badge: 'Deprecated',
```

Comment:
Could we translate the 2 "deprecated" badge references too? Would it makes sense?